packages/frontend/src/pages/MinimalDashboard.tsx

```diff
@@ -19,12 +24,18 @@ const ResponsiveGridLayout = WidthProvider(Responsive);
 
 const MinimalDashboard: FC = () => {
     const { dashboardUuid } = useParams<{ dashboardUuid: string }>();
+    const schedulerUuid = useSearchParams('schedulerUuid');
+
     const {
         data: dashboard,
         isError,
         error,
     } = useDashboardQuery(dashboardUuid);
 
+    const { data: scheduler } = useScheduler(schedulerUuid!, {
+        enabled: !!schedulerUuid,
+    });
+
     if (isError) {
```

Comment:
Non blocking issue: We need to update this condition based on error from fetching the scheduler. We could do it in a separate PR.